src/controller/DevicePairingDelegate.h

```diff
@@ -109,6 +109,12 @@ class DLL_EXPORT DevicePairingDelegate
      * in order to resume the commissioning process.
      */
     virtual void OnScanNetworksFailure(CHIP_ERROR error) {}
+
+    /**
+     * @brief
+     *  Called to check if the discover-once option is enabled.
+     */
+    virtual bool IsDiscoverOnce() { return false; }
```

Comment:
This really doesn't make sense in general on this interface.

What would make sense is some sort of notification that a device was discovered that allows the pairing delegate to say whether to proceed (possibly three options: "proceed", "skip this one", "stop").

Reply:
I tried to do a minimum change of codes to run the TC-CADMIN-1.9.
I agree with you. This interface doesn't make sense at all.
I'll find a better way to fix this.